content/tech/picnic/error-prone-support/error-prone-support/error-prone-support-0.4.0.buildspec

```diff
@@ -7,14 +7,10 @@ display=${groupId}:${artifactId}
 gitRepo=https://github.com/PicnicSupermarket/${artifactId}.git
 gitTag=v${version}
 
-# XXX: Once `rebuild.sh` supports the combination (tool=mvn-3.8.6, jdk=17) use
-# that, and drop `license:add-third-party@check-third-party-licenses` and
-# `-Dverification.skip` from the build command below.
-# See https://github.com/jvm-repo-rebuild/reproducible-central/issues/90.
-tool=mvn-3.8.5
+tool=mvn-3.8.6
```

Comment:
As of 840ca7f4c228d4a609f5db276d7c0ef63349d5d8 there's a fallback to `eclipse-temurin`, which side-steps issue #90.